server/app/views/questiontypes/QuestionBaseFragment.html

```diff
@@ -39,11 +39,15 @@
   <div
     th:id="${questionId} + '-description'"
     th:text="${question.getQuestionHelpText()}"
+    class="cf-applicant-question-help-text"
   ></div>
 </th:block>
 
 <th:block th:fragment="titleContent(question)">
-  <span th:text="${question.getQuestionText()}" class="font-sans-lg"></span>
+  <span
+    th:text="${question.getQuestionText()}"
+    class="font-sans-lg cf-applicant-question-text"
```

Comment:
non-blocking question - Can this font be added to this class?

Reply:
I looked into editing styles.css but:

- We're supposed to get new mocks soon
- I'm not sure how to will work pre- and post- North Star (do we want to tie style to this class?)
- I'm a CSS noob

So, I'd like wait to make this change